2/app.py

```diff
@@ -0,0 +1 @@
+lets make something here

```

Comment:
@coderabbitai read the readme in this 2/ dir and help me make a file to complete it